tests/specs/functional/lemmas-spec.k

```diff
@@ -504,7 +504,6 @@ module LEMMAS-SPEC
  // bitwise
  // -------
 
-    claim [bit-n005-0]: <k> runLemma ( notMaxUInt5   &Int X ) => doneLemma( 0 ) ... </k> requires #rangeUInt (   5 , X )
```

Comment:
The four lemmas with `005` have been removed as they are not representative of the slot updates, from which all of these sets of lemmas stem, and which occur only for whole bytes only.